### ISSUE TITLE
Did you use pretrained models for training all fine-grained datasets?

### ISSUE THREAD

**Issue by zzzucf** [ucbdrive/dla#3]
In your paper, figure 6 showed all results for fine-grained datasets. However, when I tried to reproduce the results in bird dataset, I can only get 53% acc. Did you use any pre-trained model for training that? And can you post the code that can be helpful to reproduce your paper's result? Best regards.

**Comment by goldentimecoolk** [comment]
I have the same problem, getting ~52% accuracy in bird test dataset. I guess data_transforms might be one tricky, because parameters including mean, std, eigval, eigvec are calculated in ImageNet dataset. Pre-train maybe another tricky. Do you improve the accuracy?

**Comment by goldentimecoolk** [comment]
> In your paper, figure 6 showed all results for fine-grained datasets. However, when I tried to reproduce the results in bird dataset, I can only get 53% acc. Did you use any pre-trained model for training that? And can you post the code that can be helpful to reproduce your paper's result? Best regards.

I have the same problem, getting ~52% accuracy in bird test dataset. I guess data_transforms might be one tricky, because parameters including mean, std, eigval, eigvec are calculated in ImageNet dataset. Pre-train maybe another tricky. Do you improve the accuracy?

**Comment by goldentimecoolk** [comment]
I have the same problem, getting ~52% accuracy in bird test dataset. I guess data_transforms might be one tricky, because parameters including mean, std, eigval, eigvec are calculated in ImageNet dataset. Pre-train maybe another tricky. Do you improve the accuracy? @zzzucf

**Comment by zzzucf** [comment]
use their pretrained mode and You can find the transform in pytorch site which is the same as their then u can repoduce the result.

**Comment by goldentimecoolk** [comment]
I used their pretrained model and got 80%+ accuracy in cub200 just now. As for transform, do you mean params they provide based on Imagenet dont need to be modified?

**Comment by zzzucf** [comment]
No need to change.

goldentimecoolk <notifications@github.com>于2019年4月11日 周四上午9:16写道：
